docs/linkchecker-external.cfg

```diff
@@ -1,6 +1,4 @@
 [filtering]
-checkextern=1
```

Comment:
Why drop external links?

Reply:
My mistake! Restoring... and I also need to make other changes.

Reply:
For comparison:

smoke-test:
```
linkchecker -f linkchecker.cfg build/html/index.html
...
That's it. 468 links in 468 URLs checked. 0 warnings found. 0 errors found.
Stopped checking at 2024-07-31 17:52:22-004 (3 seconds)
```

weekly html check (with latest changes):
```
$ linkchecker --no-warnings -f linkchecker-external.cfg build/html/index.html
...
That's it. 1431 links in 1463 URLs checked. 0 warnings found (6 ignored or duplicates not printed). 0 errors found.
Stopped checking at 2024-07-31 18:13:00-004 (1 minute, 41 seconds)
```

weekly md check:
```
$ find .. | grep '\.md$' | grep -v 'venv/\|.git/\|CHANGELOG\|vendor/\|R/opendp/README.md' | xargs linkchecker --no-warnings -f linkchecker-external.cfg
...
That's it. 71 links in 74 URLs checked. 0 warnings found (19 ignored or duplicates not printed). 0 errors found.
Stopped checking at 2024-07-31 18:18:19-004 (1 minute, 1 seconds)
```